udf_js/json_extract_events.sql

```diff
@@ -0,0 +1,85 @@
+CREATE TEMP FUNCTION
+  udf_js_json_extract_events (input STRING)
+  RETURNS ARRAY<STRUCT<
+  event_process STRING,
+  event_timestamp INT64,
+  event_category STRING,
+  event_object STRING,
+  event_method STRING,
+  event_string_value STRING,
+  event_map_values ARRAY<STRUCT<key STRING, value STRING>>
```

Comment:
Is the `event_` prefix on each of these fields added to avoid having names be keywords? It may be a reasonable compromise, but it's unfortunate to have the interface be this verbose.

Reply:
It's consistent with the column names from the events dataset (where the `event_` prefix prevented collisions with ping-level fields as well as being explicit about the source of the data for some of these fields where it might be ambiguous.) I'm open to removing the prefix in the context of the struct, although I do expect these to be exploded out fairly frequently in actual use.

Reply:
> I do expect these to be exploded out fairly frequently in actual use.

That seems like a strong argument for keeping the event_ prefix. Thanks for the context.